modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalNavTag.java

```diff
@@ -41,10 +42,30 @@ public int doStartTag() throws JspException {
 		return super.doStartTag();
 	}
 
+	public String getActive() {
+		if (_active != null) {
+			return _active;
+		}
+
+		return _getActiveVerticalNavItemKey(getVerticalNavItems());
```

Comment:
If I understood [the Clay implementation](https://github.com/liferay/clay/blob/master/packages/clay-core/src/vertical-nav/Item.tsx#L122), this is not strictly needed. It would work even we leave "active" tag attribute as undefined, with active set on the item attribute. But, this is fine as a future-proof measure, for the case that Clay drops support for [the deprecated item attribute](https://github.com/liferay/clay/blob/master/packages/clay-core/src/vertical-nav/Item.tsx#L17-L21). :+1: